src/node_zlib.cc

```diff
@@ -517,12 +523,28 @@ class ZCtx : public AsyncWrap {
         break;
       case INFLATE:
       case GUNZIP:
-      case INFLATERAW:
       case UNZIP:
         ctx->err_ = inflateInit2(&ctx->strm_, ctx->windowBits_);
         ctx->env()->isolate()
             ->AdjustAmountOfExternalAllocatedMemory(kInflateContextSize);
         break;
+      case INFLATERAW:
+        ctx->err_ = inflateInit2(&ctx->strm_, ctx->windowBits_);
+        ctx->env()->isolate()
+            ->AdjustAmountOfExternalAllocatedMemory(kInflateContextSize);
```

Comment:
Just a suggestion, but if you want to avoid duplicating this, you can just turn the `break;` statement above into `if (ctx->mode_ != INFLATERAW) break;` and leave the `case INFLATERAW:` where it is.